MetaU Capstone Project/API/SpotifyManager.m

```diff
@@ -7,6 +7,7 @@
 
 #import "SpotifyManager.h"
 #import "SpotifyiOS/SpotifyAppRemote.h"
+#import "MusicSessionViewController.h"
```

Comment:
I don't think you need to explicitly import this anymore.

Since you're using NSNotifications for delegate operations, you don't need to know which class is receiving the notification. That class simply needs to be a "listener"